<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find binocular)/config/binocular_control.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/binocular" args="joint_state_controller
    joint0_position_controller 
    joint1_position_controller 
    joint2_position_controller 
    joint3_position_controller 
    joint4_position_controller 
    joint5_position_controller "/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
    <remap from="/joint_states" to="/binocular/joint_states" />
  </node>
 
  <!-- Create node to control joint positions using joint controller 
  <node name="joint_positions_node" pkg="binocular" type="gaze.py" output="screen" respawn="true" />
 -->

</launch>